docs/conf.py

```diff
@@ -40,8 +38,8 @@ def setup(app):
 sys.path.insert(0, os.path.abspath('roman_datamodels/'))
 
 # -- General configuration ------------------------------------------------
-conf.read([os.path.join(os.path.dirname(__file__), '..', 'setup.cfg')])
-setup_cfg = dict(conf.items('metadata'))
+conf = tomli.load('../pyproject.toml')
```

Comment:
```suggestion
with open(Path(__file__).parent.parent / "pyproject.toml", "rb") as configuration_file:
    conf = tomli.load(configuration_file)
```

`from pathlib import Path` will need to be added at the top as well